content/references/examples/processing/curveDetail_/curveDetail_.pde

```diff
@@ -1,8 +1,8 @@
 void setup(){
-  size(400,400);
+  size(400,400,P2D);
 }
 
-void draw() {
+void draw(){
   curveDetail(1);
```

Comment:
Needs `background(204);` to prevent the curves drawing upon themselves.

Reply:
What is the need to chnage the background @SableRaf they look exactly the same, how is it going to prevent curves from drawing upon themselves?
I saw no changes between the images generated with or without background (204)

Reply:
Also the default background and background (204) are same

Reply:
Hi @yogitheboss. You're right that it shouldn't make a difference in theory, considering the `noLoop()` but it seems to make a difference for me on macOS 12.5.1 (Monterey) at least (haven't tested on Windows)

No background:

<img width="400" alt="image" src="https://user-images.githubusercontent.com/290261/218518328-c55b8a64-9270-4e98-afd0-e7128cd3556d.png">

With `background(204)` added:

<img width="400" alt="image" src="https://user-images.githubusercontent.com/290261/218518439-dd49aee1-087e-4523-8f6a-8ead2a6c6eab.png">

Reply:
ok @SableRaf I have added the image and added the background image you can check it now.